strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/gremlin/adapters/UserAdapter.java

```diff
@@ -67,6 +86,42 @@ private User map(Traverser<Map<String, Object>> t)
 
     @Override
     public UnfoldEntityTraversal<Vertex, Vertex> unfold(User entity)
+    {
+        String storedUserId = Vertices.USER + ":" + UUID.randomUUID().toString();
+
+        EntityTraversal<Vertex, Vertex> userRoleTraversal = __.identity();
+        EntityTraversal<Vertex, Vertex> unfoldTraversal = __.identity();
+
+        if (entity.getNativeId() != null)
+        {
+            unfoldTraversal.V(entity.getNativeId())
```

Comment:
we don't need `V(entity.getNativeId())` here. It's assumed that `unfold` method should be called when we already have vertex as current traversal element.

Reply:
Fixed.